FirebaseSwiftUI/FirebasePhoneAuthSwiftUI/Sources/Views/PhoneAuthView.swift

```diff
@@ -27,7 +27,7 @@ import SwiftUI
 @MainActor
 public struct PhoneAuthView {
   @Environment(\.dismiss) private var dismiss
-  @State private var errorMessage = ""
+  @State private var currentError: AlertError?
```

Comment:
why do we have currentError here and in authService? Is currentError meant to be view specific or app's state?

Reply:
Yeah, PhoneAuthView is initiated outside of AuthPickerView so it needs its own errorAlert and we need to track the currentError.